lib/prometheus_exporter/instrumentation/delayed_job.rb

```diff
@@ -31,6 +31,7 @@ def initialize(client: nil)
     def call(job, max_attempts, enqueued_count, pending_count, *args, &block)
       success = false
       start = ::Process.clock_gettime(::Process::CLOCK_MONOTONIC)
+      latency = Time.current - job.run_at
```

Comment:
Is it safe to call `Time.current` here because delayed job depends on active support?

Reply:
Exactly, they both handle the time in the same way, DJ uses `Time.zone` within `db_time_now` method for setting the `run_at` value.